index.html

```diff
@@ -16,25 +16,33 @@
 <!--===================================================================================-->
 
 <section id="what">
-  <h2>What can I do with it?</h2>
+  <h2>All the apps you need, all in one place.</h2>
   <ul>
-    <li class="create">
-      <h3>Create</h3>
-      <p>Create Google-Docs-like spreadsheets, documents, forms, etc. with
-        EtherCalc, Etherpad, Sandforms, Draw.io, and more.</p>
+    <li class="chat">
+      <h3>Chat</h3>
+      <p>Sync up with your colleagues securely using Rocket.Chat.</p>
     </li>
-    <li class="collaborate">
-      <h3>Collaborate</h3>
-      <p>Share documents, diagrams, and other files with your colleagues and friends, and collaborate in real-time.</p>
+    <li class="file-storage">
+      <h3>File Storage</h3>
+      <p>Store all your files in the cloud and share them with others using Davros.</p>
     </li>
-    <li class="communicate">
-      <h3>Communicate</h3>
-      <p>Sync up with your colleagues securely with great chat applications like Rocket.Chat.</p>
+    <li class="task-mgmt">
+      <h3>Task &amp; Project Management</h3>
+      <p>Organize all your tasks and ideas using Wekan.</p>
+    </li>
+    <li class="document">
+      <h3>Document Editing</h3>
+      <p>Create, edit, and collaborate on documents with Etherpad.</p>
     </li>
   </ul>
 
+  <div class="action">
+    <a href="https://oasis.sandstorm.io/demo">Demo the Productivity Suite</a>
+    <a href="/get">Get Sandstorm</a>
```

Comment:
IMHO it's weird that this says "Get Sandstorm" when the previous one says "Demo the Productivity Suite".

So I suggest instead of "Get Sandstorm" it should say, "Get it now".

This is not a show-stopper.